routes/contract.js

```diff
@@ -3,22 +3,46 @@ import {
   createContractList,
   findContractByFirstParty,
   createContractDetail,
+  updateContractList,
+  getContractContentByContractNo,
+  updateSecondPartySignature,
+  updateSecondParty,
+  findInProgressContracts,
 } from "../services/contract.js";
 
 const router = express.Router();
 
 /** /contract */
 router.get("/", async (req, res) => {
   try {
-    let contractList = await findContractByFirstParty("AMD");
-    return res.status(201).json({ status: 'success', message: 'Contract created successfully', contractList });
+    let firstParty = req.user.firstName + req.user.lastName;
+    let contractList = await findContractByFirstParty(firstParty);
+    return res.status(201).json({ status: 'success', message: 'Get contract list successfully', contractList });
```

Comment:
Can you please change this to 200, since 201 indicates that something is created in the database